docs/en/04_Changelogs/4.9.0.md

```diff
@@ -2,13 +2,38 @@
 
 ## Overview
 
+- [Security audit and regression test](#audit)
+- [For development teams of Common Web Platform projects](#cwp-end)
+- [Dropping support for Internet Explorer 11](#ie11eol)
```

Comment:
Maybe this stuff should be after "Features and enhancements"

Reply:
I think it's fine where it is